### ISSUE TITLE
menu or rating

### ISSUE THREAD

**Issue by e178551** [meanjs/mean#1777]
bower update
1) angular-bootstrap#~0.13.0 which resolved to 0.13.4 and is required by angular-rtcomm#1.0.3
2) angular-bootstrap#^2.5.0 which resolved to 2.5.0 and is required by mean
if i choose 1 the ui.bootstrap.rating is not working
if i choose 2 the menu is not working

**Comment by e178551** [comment]
Unable to find a suitable version for angular-bootstrap, please choose one by typing one of the numbers below:
    1) angular-bootstrap#~0.13.0 which resolved to 0.13.4 and is required by angular-rtcomm#1.0.3
    2) angular-bootstrap#^2.5.0 which resolved to 2.5.0 and is required by mean

Prefix the choice with ! to persist it to bower.json

? Answer 1

Please note that,
    angular-file-upload#1.1.6 depends on angular#~1.2.11 which resolved to angular#1.2.32
    angular-animate#1.3.20, angular-messages#1.3.20, angular-mocks#1.3.20, angular-resource#1.3.20 depends on angular#1.3.20 which resolved to angular#1.3.20
    mean depends on angular#~1.4 which resolved to angular#1.4.14
    angular-tree-control#0.2.28 depends on angular#~1.4.4 which resolved to angular#1.4.14
    angular-google-maps-native#2.0.0 depends on angular#1.2 - 1.4 which resolved to angular#1.4.14
    angular-bootstrap#2.5.0, angular-ui-tinymce#0.0.18 depends on angular#>=1.4.0 which resolved to angular#1.6.4
    ng-file-upload#12.1.0 depends on angular#>1.2.0 which resolved to angular#1.6.4
    angular-rtcomm#1.0.3 depends on angular#^1.4.6 which resolved to angular#1.6.4
    angularjs-geolocation#0.1.1 depends on angular#~1.x which resolved to angular#1.6.4
    angular-bootstrap#0.13.4 depends on angular#>=1.3.0 which resolved to angular#1.6.4
    angular-simple-logger#0.1.7 depends on angular#^1.2 which resolved to angular#1.6.4
    angular-ui-utils#3777381cf2 depends on angular#>= 1.0.2 which resolved to angular#1.6.4
    angular-ui-router#0.2.18 depends on angular#^1.0.8 which resolved to angular#1.6.4
    ngGeolocation#0.0.8 depends on angular#>=1.0.8 which resolved to angular#1.6.4
    angular-sanitize#1.6.4 depends on angular#1.6.4 which resolved to angular#1.6.4
    angular-google-maps#2.4.1 depends on angular#1.2 - 1.6 which resolved to angular#1.6.4
Resort to using angular#~1.3 which resolved to angular#1.3.20
Code incompatibilities may occur.

bower angular-bootstrap#~0.13.0                                 install angular-bootstrap#0.13.4

angular-bootstrap#0.13.4 public/lib/angular-bootstrap
└── angular#1.6.4

**Comment by e178551** [comment]
Unable to find a suitable version for angular-bootstrap, please choose one by typing one of the numbers below:
    1) angular-bootstrap#~0.13.0 which resolved to 0.13.4 and is required by angular-rtcomm#1.0.3
    2) angular-bootstrap#^2.5.0 which resolved to 2.5.0 and is required by mean

Prefix the choice with ! to persist it to bower.json

? Answer 2

Please note that,
    angular-file-upload#1.1.6 depends on angular#~1.2.11 which resolved to angular#1.2.32
    angular-animate#1.3.20, angular-messages#1.3.20, angular-mocks#1.3.20, angular-resource#1.3.20 depends on angular#1.3.20 which resolved to angular#1.3.20
    angular-rtcomm#1.0.3 depends on angular#^1.4.6 which resolved to angular#1.4.14
    angular-google-maps-native#2.0.0 depends on angular#1.2 - 1.4 which resolved to angular#1.4.14
    angularjs-geolocation#0.1.1 depends on angular#~1.x which resolved to angular#1.4.14
    mean depends on angular#~1.4 which resolved to angular#1.4.14
    angular-simple-logger#0.1.7 depends on angular#^1.2 which resolved to angular#1.4.14
    angular-google-maps#2.4.1 depends on angular#1.2 - 1.6 which resolved to angular#1.4.14
    angular-ui-utils#3777381cf2 depends on angular#>= 1.0.2 which resolved to angular#1.4.14
    angular-ui-router#0.2.18 depends on angular#^1.0.8 which resolved to angular#1.4.14
    angular-bootstrap#0.13.4 depends on angular#>=1.3.0 which resolved to angular#1.4.14
    ngGeolocation#0.0.8 depends on angular#>=1.0.8 which resolved to angular#1.4.14
    ng-file-upload#12.1.0 depends on angular#>1.2.0 which resolved to angular#1.4.14
    angular-tree-control#0.2.28 depends on angular#~1.4.4 which resolved to angular#1.4.14
    angular-bootstrap#2.5.0, angular-ui-tinymce#0.0.18 depends on angular#>=1.4.0 which resolved to angular#1.6.4
    angular-sanitize#1.6.4 depends on angular#1.6.4 which resolved to angular#1.6.4
Resort to using angular#~1.3 which resolved to angular#1.3.20
Code incompatibilities may occur.

bower angular-bootstrap#^2.5.0                                  install angular-bootstrap#2.5.0

angular-bootstrap#2.5.0 public/lib/angular-bootstrap
└── angular#1.4.14

**Comment by simison** [comment]
Yep, `angular-bootstrap` 2.x has a lot of breaking changes compared to 1.x and meanjs needs and upgrade on that front. PR welome!
